multiclient/client.go

```diff
@@ -513,6 +513,7 @@ func (mc *Client) SendTransaction(ctx context.Context, tx *types.Transaction) er
 //
 // `isPostToAll` is true means waiting until received all responsese of JSON-RPC calls and return error if failed to perform JSON-RPC call to all eth client.
 // Otherwise, just waiting for the first successful response and return error if failed to perform JSON-RPC call to all eth client.
+// FIXME: The `result` may be overrode to wrong value because we share a `result` in all goroutines.
```

Comment:
Please help to add same comment for `BatchCallContext`.

Reply:
OK